src/fortune-wheel-booth-frontend/src/utils/findPrize.ts

```diff
@@ -1,6 +1,11 @@
 import { PRIZES } from '../costants';
 
 export const findPrize = (option: string) => {
-  const prize = PRIZES.find((prize, index) => prize.option === option);
-  return prize ? PRIZES.indexOf(prize) : 99; // TODO: Fix this, 99 represents "no prize exists for this option"
+  if (option === 'merch') {
+    const merchNumbers = [1, 2, 3, 4, 5, 6];
+    option =
+      'merch' + merchNumbers[Math.floor(Math.random() * merchNumbers.length)];
```

Comment:
I think with Math.floor you never get the `merch6` option

Reply:
<img width="575" alt="Screen Shot 2024-05-24 at 12 16 22 PM" src="https://github.com/ilbertt/fortune-wheel-booth/assets/65824118/2d93e7bf-472b-49f7-b342-1979a457f84d">

Reply:
Woops yeah I misread the code, sorry